src/styles/blocks/card.scss

```diff
@@ -0,0 +1,97 @@
+@import "../utils/variables";
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #ffff;
+  transition: 0.3s all ease-in;
```

Comment:
1) we rarely want to transition all of the properties, this is not the right case.
2) reorder your values according to [syntax](https://developer.mozilla.org/en-US/docs/Web/CSS/transition#syntax)